robot_controllers/src/diff_drive_base.cpp

```diff
@@ -166,11 +166,7 @@ int DiffDriveBaseController::init(ros::NodeHandle& nh, ControllerManager* manage
   params_pub_.publish(limiter_.getParams());
 
   // Publish cmd values after they have been limited
-  nh.param<bool>("publish_limited_cmd", publish_limited_cmd_, false);
-  if (publish_limited_cmd_)
-  {
-    limited_cmd_pub_ = nh.advertise<geometry_msgs::Twist>("limited_cmd", 10);
-  }
+  limited_cmd_vel_pub_ = nh.advertise<geometry_msgs::Twist>("limited_cmd_vel", 10);
```

Comment:
I would suggest we actually call this "cmd_vel_limited" -- so that when you do "rostopic list" you see it near the original topic